docs/metrics.md

```diff
@@ -63,6 +63,8 @@ recorded and aggregated over fixed intervals during a perf analyzer run.
 * `cpu_used_ram`: The total amount of memory used by all CPUs
 * `cpu_available_ram`: The total amount of availble CPU memory.
 
+Warning: Collecting CPU metrics might affect performance. By default, CPU metrics are collected only if CPU(s) are used for inferencing. The collection of CPU metrics can be disabled by setting `cpu_monitor_disable` to `True`, see [Configuring Model Analyzer](docs/config.md) for details.
```

Comment:
"Warning:" looks better in bold italic _**Warning**_

Reply:
Updated.